src/tools/weather/openMeteo.ts

```diff
@@ -43,20 +47,23 @@ interface LocationSearch {
   language?: string;
 }
 
+const omittedKeys: string[] = [
```

Comment:
I would make these part of `constructor` so that a user can easily modify them without forcing us to make changes in the framework.

TIP: `string[]` -> `(keyof OpenMeteoToolResponse)[]` + add missing values to`OpenMeteoToolResponse` interface.

Reply:
Thanks for the tip. I refactored and now you can just provide a response filter based on the OpenMeteoToolResponse keys and filter anything you like.